src/backends/SDLBackend.zig

```diff
@@ -28,6 +28,16 @@ pub const initOptions = struct {
     /// content of a PNG image (or any other format stb_image can load)
     /// tip: use @embedFile
     icon: ?[]const u8 = null,
+    /// Set the minimum size of the window
+    min_size: ?struct {
+        width: u32,
+        height: u32,
+    } = null,
+    /// Set the maximum size of the window
+    max_size: ?struct {
+        width: u32,
+        height: u32,
+    } = null,
```

Comment:
I'd recommend you use the ``Size`` struct (in ``src/Size.zig``) here, and maybe even in the "normal" width/height too.

Reply:
I thought that too initially, but there's no way to ask for a fractional size right?

Reply:
But then I'd turn the `width` and `height` at the top also into a `Size` otherwise we have two distinct ways to express the window size.

Reply:
Either way is fine (pair of u32 or Size), but it probably makes sense to use the same thing for min/max/initial size.  @r4gus how about you go ahead with what you think is best, and we can change it later if needed?

Reply:
> But then I'd turn the `width` and `height` at the top also into a `Size` otherwise we have two distinct ways to express the window size.

Yes, that's the intent; this way it's uniform. However I agree with what David said, go with what you think is best.

Reply:
Ok then lets make it a `Size`